src/index.ts

```diff
@@ -16,7 +16,8 @@ Mongoose.connection
     console.log("Connected to MongoDB");
     app.emit("ready");
   })
-  .on("error", () => {
+  .on("error", (e) => {
+    console.log(e);
     console.log("Couldn't connect to DB");
```

Comment:
Cool, but let's take off the comments afterwards don't do the same mistakes I did 😊